### ISSUE TITLE
Send alert to customers when website has no visits anymore

### ISSUE THREAD

**Issue by adriaanvanrossum** [simpleanalytics/roadmap#518]
From a customer

> Since there was no traffic I was concerned so I noticed that the script was deleted in a previous push to my website.
> A good feature to consider is maybe an error message/email if the tracker stopped working.

**Comment by mstnorris** [comment]
I second this. On a couple of occasions the _production_ assets weren't built and the development versions were published causing the analytics not to be loaded.

**Comment by beeneeb16** [comment]
Isn't this feature already implemented?
![image](https://user-images.githubusercontent.com/61513680/213287250-8ffa8ad3-a11b-4e85-a9c7-7c83ac0551fa.png)

**Comment by adriaanvanrossum** [comment]
Yes, but it's a bit buggy. It doesn't work as well as intended. Will redo this, because it happens more often than customers want.
